driver/src/main/java/org/neo4j/driver/v1/Values.java

```diff
@@ -268,44 +275,64 @@ public static Value value( final Map<String,Object> val )
         return VALUE;
     }
 
-    public static Function<Value,String> valueToString()
+    public static Function<Value,String> valueAsString()
```

Comment:
Nitpicking now, and really entirely unrelated to this PR (and should not be resolved in this PR), I don't like the `valueAsXX` because it reads odd when this is used:

    value.asList( valueAsString() );

I think it'd be nicer to have it be `ofString` or `asString` here:

    value.asList( ofString() );